crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -2361,3 +2361,67 @@ ttccccC'''cC'-'Cc'''1'1cctt
      ttttttttttttttttt t
        tt   ttt ttt t
 ENDMAP
+
+###############################################################################
+# Growth, Death and Undeath, by byrel
+# 1/11 chance of an undead lernaean hydra
+NAME:    byrel_swamp_growth_death_and_undeath
+TAGS:    no_monster_gen
+PLACE:   Swamp:$
+ORIENT:  float
+TAGS:    no_rotate
+MONS:    swamp dragon / swamp drake / hydra
+MONS:	 freezing wraith / flayed ghost / phantasmal warrior / ghoul
+MONS:    hydra skeleton / swamp dragon skeleton / death drake skeleton
+MONS:    hydra zombie / swamp dragon zombie / death drake zombie
+MONS:    spectral hydra / hydra simulacrum
+MONS:    patrolling thorn hunter / patrolling shambling mangrove
+MONS:    patrolling spriggan druid god:fedhas
+KITEM:   O = decaying rune of Zot
+KFEAT:   _ = altar_fedhas
```

Comment:
`_` and `D` are symbols that don't have any obvious similarity or mnemonic. What about `F` and `Y` to label each lobe, or `_` and `-` to make them both look altar-ish?

Reply:
Fair enough